src/components/__tests__/advanced-marker.test.tsx

```diff
@@ -148,8 +148,10 @@ describe('map and marker-library loaded', () => {
       .get(google.maps.marker.AdvancedMarkerElement)
       .at(0) as google.maps.marker.AdvancedMarkerElement;
 
-    expect(marker.content?.firstChild).toHaveClass('classname-test');
-    expect(marker.content?.firstChild).toHaveStyle('width: 200px');
+    expect(marker.content?.firstChild?.firstChild).toHaveClass(
```

Comment:
wondering if we should replace this with something like "expect that there is an element with classname-test somewhere in marker.content" – feels like specifying the exact hierarchy here isn't actually what we want to test.

Reply:
Yes, that sounds like a better approach.